test/topology/test_aggregation.py

```diff
@@ -0,0 +1,80 @@
+#
+# Copyright (C) 2025-present ScyllaDB
+#
+# SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
+#
+import asyncio
+import pytest
+import time
+import logging
+import random
+
+from cassandra.cluster import NoHostAvailable  # type: ignore
+
+from test.pylib.manager_client import ManagerClient
+from test.pylib.rest_client import inject_error
+from test.pylib.util import wait_for, wait_for_cql_and_get_hosts
+from test.topology.conftest import skip_mode
+
+logger = logging.getLogger(__name__)
+
+@pytest.mark.asyncio
+@skip_mode("release", "error injections are not supported in release mode")
+async def test_cancel_mapreduce(manager: ManagerClient):
+    """
+    This test verifies that stopping the supercoordinator of a mapreduce task cancels
+    outgoing queries to other nodes, which would otherwise prevent the shutdown.
+    """
+
+    running_servers = await manager.running_servers()
+    assert len(running_servers) > 0
+
+    s1 = await manager.server_add(cmdline=["--logger-log-level", "forward_service=trace"])
+    s2 = running_servers[0]
+    assert s1.server_id != s2.server_id
```

Comment:
I'm pretty sure there is a way to change the log level in runtime. The `topology` suite already provides more than one node, so instead of adding a new node you could just assert that there are at least two nodes and change the log level of one of them.

IIRC it's the `/storage_service/logging_level` endpoint (see `api/api-doc/storage_service.json`)